src/azure-cli/azure/cli/command_modules/network/commands.py

```diff
@@ -187,7 +187,7 @@ def load_command_table(self, _):
     network_private_endpoint_sdk = CliCommandType(
         operations_tmpl='azure.mgmt.network.operations#PrivateEndpointsOperations.{}',
         client_factory=cf_private_endpoints,
-        min_api='2018-08-01'
+        min_api='2020-06-01'
```

Comment:
Will upgrade this API version break user?

Reply:
this was update to fix az network private-endpoint module load issue. with min_api='2018-08-01' it fails with the below error: 
ValueError: The operation 'azure.mgmt.network.v2018_11_01.operations#PrivateEndpointsOperations.get' is invalid.

let me know if this fix is not right? 

@Juliehzl FYI

Reply:
if thats the case, it should have been failing earlier as well? The error indicates that PrivateEndpointOperations is not avaialabe in 2018-11-01 api version

Reply:
That's exactly what I meant
> The error indicates that PrivateEndpointOperations is not avaialabe in 2018-11-01 api version

Please make sure that 2020-06-01 is the correct version that would enable PrivateEndpoint.